runtime/wasm/jslib.wat

```diff
@@ -414,47 +414,6 @@
          (call $wrap_fun_arguments
             (call $wrap_callback_strict (i32.const 1) (local.get 0)))))
 
-   (func (export "caml_callback")
```

Comment:
I think this was deadcode, because generate.ml was aliasing `caml_callback` to `caml_trampoline`. @vouillon, can you confirm this is correct ?

Reply:
This function is called from `runtime/wasm/runtime.js`.

It might be clearer to use `caml_trampoline` in `compiler/lib/effects.ml`, and alias it to `caml_callback` when generating JavaScript code.

Reply:
See #1922